static/app/components/indicators.tsx

```diff
@@ -54,15 +52,19 @@ class Indicators extends React.Component<Props> {
   }
 }
 
-const IndicatorsContainer = createReactClass<Omit<Props, 'items'>>({
-  displayName: 'IndicatorsContainer',
-  mixins: [Reflux.connect(IndicatorStore, 'items') as any],
+type State = {
+  items: Indicator[];
+};
+
+class IndicatorsContainer extends React.Component<Omit<Props, 'items'>, State> {
+  state: State = {items: IndicatorStore.get()};
+  componentWillUnmount() {
+    this.unlistener?.();
+  }
 
-  getInitialState() {
-    return {
-      items: [],
-    };
-  },
+  unlistener = IndicatorStore.listen((items: Indicator[]) => {
+    this.setState({items});
+  }, undefined);
```

Comment:
oh this is nice, we should definitely change all of our reflux usage to this type of pattern and kill the mixins if we can.